src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -720,7 +720,9 @@
 
   <ItemGroup Condition=" '$(UsesHelixSdk)' == 'true' ">
     <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
-    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" />
+
+    <!-- Browser-Wasm follows a very different workflow, which is currently out of scope of the Log Checker and Test Watcher. -->
+    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" Condition="'$(TargetsBrowser)' != 'true'" />
```

Comment:
We're not going to use the watcher on any platforms that use XHarness, so don't include it there either.

```suggestion
    <!-- Browser-Wasm follows a very different workflow, which is currently out of scope of the Log Checker and Test Watcher. -->
    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" Condition="'$(TargetsBrowser)' != 'true' and '$(TargetHasHelixXHarnessSdkSupport)' != 'true'" />
```

Reply:
I think this would be better suited for the log checker PR. Since we are passing the watcher to Helix through the Core_Root, then this is no longer related here. I'll update the comment, removing the mention to the Test Watcher.